src/MobileList.js

```diff
@@ -0,0 +1,129 @@
+        { dayStr }{' | '}{ timeStrs }
+      </p>
+    )
+  }
+
+  if (sortedDates.length <= MAX_DAYS_IN_LIST) {
+    return sortedDates.map(dateRowFactory)
+  } else {
+    let nextDay = sortedDates[MAX_DAYS_IN_LIST - 1];
+    let lastDay = sortedDates[sortedDates.length - 1];
+    let nextStart = sortedTimesByDate[nextDay][0].start;
+    let lastStart = sortedTimesByDate[lastDay][0].start;
+    return sortedDates.slice(0,MAX_DAYS_IN_LIST - 1).map(dateRowFactory).concat(
+      <p>
+        More Times from {nextStart.twix(lastStart, { allDay : true }).format()}
+      </p>
+    )
+  }
+}
+
+export function MobileList(props){
+
+  //Mobile's location filter doesn't filter but moves the currentIndex to the location's first event
+  useEffect(() => {
+
+    //Location filter
+    if(props.locFilt != null){
+
+      for(let x = 0; x < props.events.length; x++) {
+        let event = props.events[x];
+        if(!('location' in props.events[props.cardIndex]) || !('location' in event['location']) || props.events[props.cardIndex]['location']['location']['latitude'] !== props.locFilt['lat'] || props.events[props.cardIndex]['location']['location']['longitude'] !== props.locFilt['lng']){
+          if('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']){
+            if(event['location']['location']['latitude'] === props.locFilt['lat'] && event['location']['location']['longitude'] === props.locFilt['lng']){
+              props.updateCardIndex(x);
+              x = props.events.length;
+            }
+          }
+        }
+      }
+    }
+  }, [props.locFilt])
+
+
+
+  const listEvents = props.events.map((event, index) => {
+
+    // Normalize Mobilize's time formatting into
+    // easy-to-use moments
+    let rawTimes = event['timeslots'].map((timeslot) => {
+      let start = moment(timeslot.start_date * 1000);
+      let end = moment(timeslot.end_date * 1000);
+      return {
+        start, end,
+        range: start.twix(end)
+      }
+    })
+
+    return (
+      <a href={event['browser_url']}
+        className="eventCard"
+        target="_blank"
+        rel="noopener noreferrer"
+        key={event['id']}
+        coord={('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']) ? "" + event['location']['location']['latitude'] + "&" + event['location']['location']['longitude'] : ""}
+        onMouseEnter={(event) => { props.updatedHover(event['currentTarget'].getAttribute('coord')) }}
+        onMouseLeave={(event) => { props.updatedHover(null) }}>
+        <div className="mobileInfo">
+          <h3>{event['title']}</h3>
+          <p><strong>{event['location']['venue']}</strong> in <strong>{event['location']['locality']}</strong></p>
+          <EventTimes rawTimes={rawTimes} />
+          <p className="eventRSVP">Click to RSVP</p>
+        </div>
+
+      </a>
+
+    )
+  }).filter((arrItem) => {
+    return arrItem != null;
```

Comment:
There are a lot of != null checks going on. 

That always worries me because JS vars tend to be undefined occasionally. Is there an unintentional distinction being made between null and undefined? Would _.isNil work, or are you relying on the weird behavior of != vs !==? The double vs single inequality is always a bit confusing to me and seems to make typo bugs easier